catalog/core/catalog-core-standardframework/pom.xml

```diff
@@ -174,7 +175,6 @@
                             solr-core,
                             solr-solrj,
                             solr-factory,
-                            commons-io,
                             commons-lang,
                             joda-time,
                             lucene-core,
```

Comment:
Why did you not have to add lucene-join to this pom or the following two poms?

https://github.com/codice/ddf/blob/master/catalog/ui/search-ui/search-endpoint/pom.xml#L188-L222
https://github.com/codice/ddf/blob/master/catalog/solr/catalog-solr-external-provider/pom.xml#L72-L105